resources/gen-ai/aws-web-crawler-container/scripts/download_files.py

```diff
@@ -0,0 +1,320 @@
+import os
```

Comment:
copyright & license comment.

Reply:
fixed in https://github.com/awslabs/generative-ai-cdk-constructs/pull/474/commits/94869bc149e4ed35dee85ff8a903754189564663